test/variable.jl

```diff
@@ -467,6 +467,30 @@ function test_variable_symmetric(ModelType)
     @test y[1, 2] === y[2, 1]
 end
 
+function test_variable_skewsymmetric(ModelType)
+    model = ModelType()
+
+    @variable model x[1:2, 1:2] in SkewSymMatrixSpace()
+    @test x[1, 2] == -x[2, 1]
+    @test iszero(x[1, 1])
+    @test iszero(x[2, 2])
+    @test model[:x] === x
```

Comment:
Add test for the number of variables actually added to the inner model

Reply:
I'm not sure how to access those?

Reply:
https://www.juliaopt.org/JuMP.jl/dev/variables/#JuMP.num_variables